server/RemoteHubClient.ts

```diff
@@ -190,4 +199,29 @@ export default class RemoteHubClient extends BaseRemoteClient {
       else callback(null);
     });
   };
+
+  private onImportProject =  (data: ImportProjectData) => {
+
+  fs.writeFile(path.join(this.server.projectsPath, "project.zip"), data.data as string, (err) => {
+    if (err) return console.log(err);
+
+    fs.createReadStream(path.join(this.server.projectsPath, "project.zip"))
+      .pipe(unzip.Parse())
+      .on("entry", (entry: any) => {
+        if (entry.type === "Directory") {
+          fs.mkdir(path.join(this.server.projectsPath, entry.path));
+        }
+        else {
+          entry.pipe(fs.createWriteStream(path.join(this.server.projectsPath, entry.path)));
+        }
+      })
+      .on("close", () => {
+        this.server.loadProject(data.name, (err: Error) => {
+          if (err) {
+            console.log(err);
+          }
+        });
```

Comment:
Regarding https://github.com/superpowers/superpowers-core/issues/46#issuecomment-250683770:

I think you'll need to change `loadProject` in `ProjectHub.ts` to pass you the newly initialized project server as a second argument, then you can access the manifest (which contains name, description, etc. to send it to clients for display in the hub).

For `actualIndex`, I think after loading the project, we'll need to add the ProjectHub's `data.projects` so that it shows up in the list, and that will give you that index. If that's too confusing I can take care of it.